### PR TITLE
feat(Backdrop): Implements backdrop (#255)

### PR DESCRIPTION
**Pull request by srambach**
Implements the backdrop that will darken and blur the background behind a modal. 
This uses `backdrop-filter`, which has very limited support in browsers. It will gracefully degrade to a dark backdrop without the blur.

### CODE REVIEW

File: src/patternfly/components/Backdrop/styles.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+.pf-c-backdrop {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: var(--pf-c-backdrop--ZIndex);
+  width: 100%;
+  height: 100%;
+  background-color: var(--pf-c-backdrop--Color);
+  // Vendor prefix needed for Safari, but it is not getting added in the autoprefixer right now
+  //-webkit-backdrop-filter: var(--pf-c-backdrop--BackdropFilter);
```
Comment (andresgalante) [inline src/patternfly/components/Backdrop/styles.scss]: If autorpefixer is not prefixing it, I doesn't hurt to uncomment line 18

File: src/patternfly/components/Backdrop/styles.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@import "../../patternfly-utilities";
+
+:root {
+  --pf-c-backdrop--ZIndex: var(--pf-global--ZIndex--xl);
+  --pf-c-backdrop--Color: rgba(0, 0, 0, .5);
```
Comment (andresgalante) [inline src/patternfly/components/Backdrop/styles.scss]: Can we add a comment about this value? I think we'll generate this as a global variable when we implement #266

File: src/patternfly/components/Backdrop/styles.scss (hunk 1 above)
Comment (srambach) [reply src/patternfly/components/Backdrop/styles.scss]: It then fails to build because of the linter.

**Comment by srambach** [comment]
Let's remember `overscroll-behavior` and add it as appropriate on the modal or other components once we decide how to handle scrolling there. It doesn't quite work they way you want it to for the backdrop since it only traps the scroll when the element itself has something to scroll.

**Pull request merged**